content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -388,6 +390,8 @@ TARGET_VERSION=2.2.1
 STORAGE_MEDIA_TYPE=application/json
 ```
 
+** Note: this upgrade procedure is for getting from 2.x to 3.x.  Version `3.0.17` is not recommended for running in production (see [prerequisites](#prereqisites) for minimum recommended etcd versions.
```

Comment:
Mark this up as an actual note.

Reply:
This note needs to be formatted correctly and with a shortcode:
```
<<{ note }>>
**Note:** This procedure upgrades from 2.x to 3.x.  Version `3.0.17` is not recommended for running in production (see [prerequisites](#prerequisites) for minimum recommended etcd versions.
<<{ /note }>>
```
So:
> <<{ note }>>
> **Note:** This procedure upgrades from 2.x to 3.x.  Version `3.0.17` is not recommended for running in production (see [prerequisites](#prerequisites) for minimum recommended etcd versions.
> <<{ /note }>>